examples/js_outs/BUILD.bazel

```diff
@@ -0,0 +1,23 @@
+"""Simple use case for swc: transpiling TypeScript using the `swc` rule
+
+Note that this example also depends on the setup in /WORKSPACE at the root of this repository.
+"""
+
+load("@aspect_bazel_lib//lib:write_source_files.bzl", "write_source_files")
+load("@aspect_rules_swc//swc:defs.bzl", "swc")
+
+# Runs `swc in.ts > ../../bazel-bin/examples/simple/in.js`
+# You can run `bazel build --subcommands //examples/simple:compile`
+# to see the exact command line Bazel runs.
+swc(
+    name = "compile",
+    srcs = ["in.js"],
+)
+
+# Assert that the output of "compile" rule matches the expected file.
+write_source_files(
+    name = "test",
+    # This is a pre-declared output of the "compile" rule, so we can refer to it directly using a Bazel label
+    # even though the file itself is generated by Bazel in ../../bazel-bin/examples/simple/in.js
+    files = {"expected.js": ":in.js"},
```

Comment:
I think this should be `:compile` to be sure you're talking about the output and not the input here